index.html

```diff
@@ -44,19 +44,32 @@ <h1><i class="snes-jp-logo brand"></i>NES.css</h1>
 
     <section class="nes-container with-title">
       <h2 class="title">Buttons</h2>
-      <div>
+      <div id="buttons">
         <button type="button" class="nes-btn">Normal</button>
         <button type="button" class="nes-btn is-primary">Primary</button>
         <button type="button" class="nes-btn is-success">Success</button>
         <button type="button" class="nes-btn is-warning">Warning</button>
         <button type="button" class="nes-btn is-error">Error</button>
         <button type="button" class="nes-btn is-disabled">Disabled</button>
       </div>
+      <button type="button" class="nes-btn is-error show-code" id="show-buttons-code"><></button>
+      <span class="nes-balloon from-right copied" id="copied-buttons">Copied!</span>
+    </section>
+    <section class="nes-container code" id="code-buttons">
+      <code>
+        <p><span><</span>button type="button" class="nes-btn">Normal<span><</span>/button></p>
```

Comment:
Is this correct HTML? 🤔 
I think it is better to use encoded code. 💻 

```html
<button type="button" class="nes-btn">Normal</button>
```
↓↓↓↓
```html
&lt;button type=&quot;button&quot; class=&quot;nes-btn&quot;&gt;Normal&lt;/button&gt;
```

Reply:
The `&quot;` is going to take me more time to do it, i can't figure out a good regex to do it, so I think I will have to change it all manually :smile:

Reply:
You can easily encode using the following tools: 🛠 

* https://www.freeformatter.com/html-escape.html
* https://webtools.dounokouno.com/htmlescape/
* https://codebeautify.org/html-escape-unescape
* etc...

Reply:
Thank you for this.